code/stable/glassbr/GlassBR_SRS.tex

```diff
@@ -577,7 +577,7 @@ \subsubsection{Data Constraints}
 Var & Physical Constraints & Software Constraints & Typical Value & TU
 \\
 \midrule
-$a$ & $a>0$ and $a>b$ & ${d_{min}}\leq{}a\leq{}{d_{max}}$ and $a<{AR_{max}} b$ & $1.5$ m & 10.0\%
+$a$ & $a>0$ and $a\geqb$ & ${d_{min}}\leq{}a\leq{}{d_{max}}$ and $a<{AR_{max}} b$ & $1.5$ m & 10.0\%
```

Comment:
I don't think this will compile in LaTeX.  I believe there will be an error when LaTeX tries to interpret the command \geqb.

Reply:
I realized this - this was fixed in c145a79